info.plist

```diff
@@ -105,6 +105,8 @@
 				<false/>
 				<key>alfredfiltersresultsmatchmode</key>
 				<integer>2</integer>
+				<key>argumenttreatemptyqueryasnil</key>
```

Comment:
I copied the updated `info.plist` file generated by Alfred so that's why you might see some unrelated changes to the `info.plist` file.